src/components/clusterDetail/ClusterProperties.tsx

```diff
@@ -14,21 +14,26 @@ const ClusterProperties: React.FC<ClusterPropertiesProps> = ({ cluster }) => (
         <Text component="h2">Cluster Details</Text>
       </TextContent>
     </GridItem>
-    <GridItem md={6} lg={6} xl={6}>
+    <GridItem md={6} lg={4} xl={4}>
       <DetailList>
         <DetailItem title="OpenShift version" value={cluster.openshiftVersion} />
         <DetailItem title="Base DNS domain" value={cluster.baseDnsDomain} />
         <DetailItem title="API virtual IP" value={cluster.apiVip} />
         <DetailItem title="Ingress virtual IP" value={cluster.ingressVip} />
       </DetailList>
     </GridItem>
-    <GridItem md={6} lg={6} xl={6}>
+    <GridItem md={6} lg={4} xl={4}>
       <DetailList>
         <DetailItem title="Cluster network CIDR" value={cluster.clusterNetworkCidr} />
         <DetailItem title="Cluster network host prefix" value={cluster.clusterNetworkHostPrefix} />
         <DetailItem title="Service network CIDR" value={cluster.serviceNetworkCidr} />
       </DetailList>
     </GridItem>
+    <GridItem md={6} lg={4} xl={4}>
+      <DetailList>
+        <DetailItem title="UUID" value={cluster.id} />
```

Comment:
We need to make sure, the user understands that this is just the `Assisted Installed UUID` and not the OCP ID which the user will use any farther.
There is a plan/wish to expose this second (probably more important for the user) ID too, but it's so far missing in the API.

@nirfarkas @andybraren

Reply:
I've discussed this briefly with @andybraren yesterday. I am calling the ID just UUID to imply that it is the cluster resource ID and when it is available, we could call the OCP ID 'OpenShift Cluster ID'. This should provide enough differentiation, what do you think?

Reply:
We can call those who understand the difference "Advanced Assisted Installer" users.